lib/spack/docs/basic_usage.rst

```diff
@@ -1196,9 +1041,41 @@ of module files:
        """Set up the compile and runtime environments for a package."""
        pass
 
-"""""""""""""""""
+
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Regenerating Module files
+^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
It was a spurious "Regenerating Module files" header.  I think all the
sections are there, but some might be edited a bit.


On Thu, Sep 8, 2016 at 1:30 AM, Massimiliano Culpo <notifications@github.com
> wrote:

> In lib/spack/docs/basic_usage.rst:
>
> > @@ -1196,9 +1041,41 @@ of module files:
> >         """Set up the compile and runtime environments for a package."""
> >         pass
> >
> > -"""""""""""""""""
> > +
> > +^^^^^^^^^^^^^^^^^^^^^^^^^
> > +Regenerating Module files
> > +^^^^^^^^^^^^^^^^^^^^^^^^^
>
> @adamjstewart <https://github.com/adamjstewart> It seems that the
> sections added here b215b19
> <https://github.com/LLNL/spack/commit/b215b19cae1eaf665dfbec4e20c36497c57dd182>
> got partially erased at some point.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/LLNL/spack/pull/1676/files/c9e83c62a002c42852fcb54a3b45d54a68cb8ccc#r77947919>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AB1cd2YPD2Khp05GHQIJ705UmJaBFpMnks5qn51ngaJpZM4JxOaa>
> .
>